<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Envio - Thermostat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="TempLabel">
         <property name="styleSheet">
          <string notr="true">font-family: &quot;Montserrat&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
border: 0;
padding: 10px;
padding-top: -15px;
color: #b71c1c;
font-size: 16px;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LumLabel">
         <property name="styleSheet">
          <string notr="true">font-family: &quot;Montserrat&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
width: 100%;
border: 0;
padding: 10px;
padding-top: -15px;
color: #0288d1;
font-size: 16px;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="OpacLabel">
         <property name="styleSheet">
          <string notr="true">font-family: &quot;Montserrat&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
width: 100%;
border: 0;
padding: 10px;
color: #4d4d4d;
font-size: 12px;

</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="TempEditButton">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	font-family:&quot;Montserrat&quot;, sans-serif;
	text-transform: uppercase;
	background: #D9534F;
	padding: 10px;
	color: #FFFFFF;
	font-size: 12px;
	outline: 0;
	border: 0;
}

QPushButton:hover
{
	font-family:&quot;Montserrat&quot;, sans-serif;
	text-transform: uppercase;
	background: #b71c1c;
	padding: 10px;
	color: #FFFFFF;
	font-size: 12px;
	outline: 0;
	border: 0;
}</string>
         </property>
         <property name="text">
          <string>Température</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LumEditButton">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	font-family:&quot;Montserrat&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #0288d1;
	border: 0;
	padding: 10px;
	color: #FFFFFF;
	font-size: 12px;
}

QPushButton:hover
{
	font-family:&quot;Montserrat&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #2e6da4;
	border: 0;
	padding: 10px;
	color: #FFFFFF;
	font-size: 12px;
}</string>
         </property>
         <property name="text">
          <string>Luminosité</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OpacEditButton">
         <property name="text">
          <string>Vitres</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="tempTxtlabel">
         <property name="styleSheet">
          <string notr="true">font-family: &quot;Montserrat&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
width: 100%;
border: 0;
padding: 10px;
color: #4d4d4d;
font-size: 12px;

</string>
         </property>
         <property name="text">
          <string>Température</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LumTxtLabel">
         <property name="styleSheet">
          <string notr="true">font-family: &quot;Montserrat&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
width: 100%;
border: 0;
padding: 10px;
color: #4d4d4d;
font-size: 12px;

</string>
         </property>
         <property name="text">
          <string>Luminosité</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="opacTxtLabel">
         <property name="styleSheet">
          <string notr="true">font-family: &quot;Montserrat&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
width: 100%;
border: 0;
padding: 10px;
color: #4d4d4d;
font-size: 12px;

</string>
         </property>
         <property name="text">
          <string>Opacité</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="PlanningEditButton">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	font-family:&quot;Montserrat&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #5cb85c;
	border: 0;
	padding: 10px;
	color: #FFFFFF;
	font-size: 12px;
}

QPushButton:hover
{
	font-family:&quot;Montserrat&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #449d44;
	border: 0;
	padding: 10px;
	color: #FFFFFF;
	font-size: 12px;
}</string>
         </property>
         <property name="text">
          <string>Planning</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ConfigEditButton">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	font-family:&quot;Montserrat&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #5cb85c;
	border: 0;
	padding: 10px;
	color: #FFFFFF;
	font-size: 12px;
}

QPushButton:hover
{
	font-family:&quot;Montserrat&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #449d44;
	border: 0;
	padding: 10px;
	color: #FFFFFF;
	font-size: 12px;
}</string>
         </property>
         <property name="text">
          <string>Configuration</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="DateLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-family: &quot;Montserrat&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
border: 0;
padding: 5px;
color: #4d4d4d;
font-size: 12px;

</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="roomLabel">
         <property name="styleSheet">
          <string notr="true">font-family: &quot;Montserrat&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
border: 0;
padding: 10px;
color: #4d4d4d;
font-size: 12px;

</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
